ports/windows/mpconfigport.h

```diff
@@ -227,14 +227,11 @@ typedef long mp_off_t;
 #include "sleep.h"
 
 #if MICROPY_ENABLE_SCHEDULER
-// Use 1mSec sleep to make sure there is effectively a wait period:
+// Use minimum 1mSec sleep to make sure there is effectively a wait period:
 // something like usleep(500) truncates and ends up calling Sleep(0).
-#define MICROPY_EVENT_POLL_HOOK \
-    do { \
-        extern void mp_handle_pending(bool); \
-        mp_handle_pending(true); \
-        msec_sleep(1.0); \
-    } while (0);
+#define MICROPY_INTERNAL_WFE(TIMEOUT_MS) msec_sleep(MAX(1.0, (double)(TIMEOUT_MS)))
+#else
+#define MICROPY_INTERNAL_WFE(TIMEOUT_MS) /* No-op */
```

Comment:
Not completely sure what MICROPY_INTERNAL_WFE is supposed to do, but scheduler.c has `MICROPY_INTERNAL_WFE(-1)` so that does not seem to work as intended?

Reply:
MICROPY_INTERNAL_WFE is "wait for event".  But it can be conservative and return immediately, but in that case it'll use more power/CPU.

If the argument is -1 it means "wait indefinitely for the next event".  Here that would just sleep 1ms.

When MICROPY_ENABLE_SCHEDULER is disabled, I don't think this macro is ever used, so doesn't need to do anything in that case.